pyproject.toml

```diff
@@ -13,24 +13,12 @@ skip = [
 minversion = "6.0"
 addopts = "--doctest-modules"
 
-testpaths = [
-    "CreeDictionary"
-]
-
 env = [
     "USE_TEST_DB=true"
 ]
 
 # pytest-django
-DJANGO_SETTINGS_MODULE = "CreeDictionary.settings"
-
-
-[project_paths]
-# A place to store paths to different components in our project.
-# Here they are all written relative to project root.
-# we have get_project_path function in CreeDictionary.utils.project_path
-# module that resolves to these paths
-# TODO: use `pip install project-paths` (requires Python 3.8+)
+DJANGO_SETTINGS_MODULE = "CreeDictionary.CreeDictionary.settings"
 
-# the file we write documentation for directories
-dir_doc_file = "docs/directory-structure.md"
+django_find_project = false
+python_paths = "."
```

Comment:
Could you please add a comment about why this was added? I think it has something to do with the new pytest-pythonpath dependency?

Reply:
I have added a comment that explains a bit but it seems like setting sys.path is just something that pipenv/pytest/mypy all handle badly.

This page I just found leads me to believe the problems I was running into *might* by fixed by ensuring we have `__init__.py` files in all the directories that need them, and only the ones that need them: 
https://docs.pytest.org/en/documentation-restructure/background/pythonpath.html

I haven’t been spending much time on this at the moment because I expect it’ll all have to change soon anyway when there are multiple `manage.py` scripts and associated django projects to test together.